src/css/profile/mobile/common/popup.less

```diff
@@ -93,6 +88,144 @@
 	}
 }
 
+@media (min-width: 0px) and (max-width: 479px) {
+	@media (min-height: 600px) and (max-height: 959px) {
+		.ui-popup {
+			width: 100%;
+			margin: 0 auto;
+			background-color: rgba(255, 255, 255, 0);
+			margin-top: 40px;
```

Comment:
How about change px ->* px_base ?
In the tizen.less , it defined by @px_base: 1px;

Reply:
I fixed.

done.